controllers/authController.js

```diff
@@ -0,0 +1,33 @@
+const User = require('../models/user');
```

Comment:
```suggestion
const User = require('../models/userResgistration');
```
If you delete `user.js` and then change this, it should be better?
I'll still have to fully test this as well